### PR TITLE
Improve the license UI and messaging.

### PR DESCRIPTION
**Pull request by nkolev92**
## Bug
Fixes: https://github.com/NuGet/Home/issues/7462, part of https://github.com/NuGet/Home/issues/7467
Regression: No  
If Regression then when did it last work:   
If Regression then how are we preventing it in future:   

## Fix
Details: 
Some strings changes based on the feedback from Karan. 

Earlier in 15.8 in the details pane for the license field we showed a clickable license url. Per Karan's feedback, changing that to View License, which is the equivalent of what we do for embedded files. 
The benefit is that an ugly url is not shown there. The con is that you don't know the exact link.

From https://github.com/NuGet/Home/issues/7467 the following are fixed.

The whitespace issues with license expressions are fixed.
The weight of the header is the same as the text on the left in the same row. 
The license file window appears at the middle of the screen. 
The hyperlinks display a tooltip. 

viewlicense
fixedmargins
tooltip

//cc @karann-msft @rrelyea @anangaur 

## Testing/Validation
Tests Added: No  
Reason for not adding tests:  Strings changes. 
Validation done:

### CODE REVIEW

**Comment by anangaur** [comment]
Nit: “View License” and not “ViewLicense”

**Comment by nkolev92** [comment]
@anangaur It is "View License" in the code, I just missed the space in the description :)
🤣

File: src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx [hunk 1]
```diff
@@ -140,7 +141,7 @@
     <comment>0 - the license expression, 1 further details</comment>
   </data>
   <data name="InvalidLicenseExpressionVersion" xml:space="preserve">
-    <value>The version string '{0}' for the License Expression is invalid.</value>
+    <value>The version string '{0}' PackageLicenseExpressionVersion is invalid.</value>
```
Comment (jainaashish) [inline src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx]: `PackageLicenseExpressionVersion '{0}' version string is invalid.`

**Review by jainaashish — APPROVED** [review]

File: src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx [hunk 2]
```diff
@@ -130,7 +130,8 @@
     <comment>{0} is the file being packed.</comment>
   </data>
   <data name="InvalidLicenseCombination" xml:space="preserve">
-    <value>Invalid metadata. Cannot specify both a License Expression and a License File.</value>
+    <value>Invalid metadata. Cannot specify both a PackageLicenseExpression and a PackageLicenseFile.</value>
+    <comment>Please don't localize PackageLicenseUrl, PackageLicenseFile and PackageLicenseExpression.</comment>
```
Comment (PatoBeltran) [inline src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx]: nit: `PackageLicenseUrl` is not used in this string, remove it from the comment.

File: src/NuGet.Core/NuGet.Packaging/Strings.resx [hunk 3]
```diff
@@ -786,27 +786,16 @@ Valid from:</comment>
...
-    <comment>0 - nupkg metadata file path
-1 - error message</comment>
-  </data>
-  <data name="DefaultError_EmptyAllowList" xml:space="preserve">
-    <value>A list of trusted signers is required but none was found.</value>
-  </data>
-  <data name="DefaultError_NoMatchInAllowList" xml:space="preserve">
-    <value>The package signature certificate fingerprint does not match any certificate fingerprint in the allow list.</value>
+  <data name="NuGetLicenseExpression_NonStandardIdentifier" xml:space="preserve">
+    <value>The license identifier(s) {0} is(are) not recognized by the current toolset.</value>
```
Comment (PatoBeltran) [inline src/NuGet.Core/NuGet.Packaging/Strings.resx]: nit: `'{0}'`

**Review by PatoBeltran — APPROVED** [review]

**Pull request merged**